### PR TITLE
Calendar invites #619

### PR DESCRIPTION
**Pull request by mattkrick**
@ackernaut can you help with the copy on the Summary Email? Doesn't look quite right by just adding the icons.

Test:
- [ ] create a new account, run through the first meeting, see the add to calendar sections in the meeting summary
- [ ] click on the google & iCalendar icons & add the meeting to your calendar. verify that the start/stop time is about when your meeting started
- [ ] do the same thing with the meeting summary that arrives in your inbox. make sure the formatting is good

### CODE REVIEW

File: src/universal/modules/email/helpers/makeCalendarInvites.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+SEQUENCE:0
+TRANSP:OPAQUE
+DTEND:${endTime}
+LOCATION:${meetingUrl}
+SUMMARY:Star Wars Day Party
+CLASS:PUBLIC
+SUMMARY:Action Meeting for ${teamName}
+CLASS:PUBLIC
+DTSTAMP:${startTime}
+RRULE:FREQ=WEEKLY;COUNT=8
```
Comment (mattkrick) [inline src/universal/modules/email/helpers/makeCalendarInvites.js]: create 8 calendar events, which should be enough to ride the free trial wave

**Comment by ackernaut** [comment]
- [ ] Convert FA icons to PNG
- [ ] UI style polish

**Comment by ackernaut** [comment]
Here’s what latest looks like, in browser. Was very similar in Gmail using Chrome.

screenshot 2017-04-19 17 51 09

**Comment by mattkrick** [comment]
calling this one good & merging into master so we can release tonight

**Pull request merged**